iam_assumerole/README.md

```diff
@@ -65,3 +65,4 @@ module "billing-assumerole" {
    - `policy_name` - **string**: Name of the IAM policy to be created.
    - `policy_description` - **string**: Description of the IAM policy.
    - `policy_document` - **list(object)**: List of Statements included in the policy document. Each _object_ in the list should include the contents of a Statement, i.e. the `sid`, `effect`, `actions`, and `resources`.
+- `permissions_boundary_policy_arn` - **string**: (REQUIRED) ARN of an existing IAM policy (from another module/source) which will be used as the Permissions Boundary for the IAM role. CANNOT be left blank; policy MUST exist outside of this module!
```

Comment:
Since this value can be left blank, this can now be shortend to match the description in `main.tf`:
```suggestion
- `permissions_boundary_policy_arn` - **string**: (REQUIRED) ARN of an existing IAM policy (from another module/source) which will be used as the Permissions Boundary for the IAM role.